release/ui/ui.go

```diff
@@ -0,0 +1,87 @@
+package ui
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/google/go-github/v39/github"
+	ecmConfig "github.com/rancher/ecm-distro-tools/cmd/release/config"
+	"github.com/rancher/ecm-distro-tools/release"
+	"github.com/rancher/ecm-distro-tools/repository"
+	"golang.org/x/mod/semver"
+)
+
+const (
+	uiOrg           = "rancher"
+	uiRepo          = "ui"
+	uiImagesBaseURL = "https://github.com/" + uiOrg + "/" + uiRepo + "/releases"
+)
+
+func CreateRelease(ctx context.Context, client *github.Client, r *ecmConfig.UIRelease, opts *repository.CreateReleaseOpts, rc bool) error {
+	fmt.Println("validating tag")
```

Comment:
I am not opposed to logging here. 
However, I remember some reviews I got telling me not to log anything at this level unless using the `debug` flag.

Reply:
Removed them, if the tag doesn't pass the validation, the return error will "log" that.